ios/Runner/Info.plist

```diff
@@ -2,6 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+    <key>LSApplicationQueriesSchemes</key>
+    <array>
+      <string>https</string>
+      <string>http</string>
+    </array>
```

Comment:
can you say sth about that?

Reply:
the library [url_launcher](https://pub.dev/packages/url_launcher) has a method `canLaunch(url)` method which using `canOpenURL(_:)` in native iOS code. To use that method we have to declare supported scheme by `LSApplicationQueriesSchemes`
[https://developer.apple.com/documentation/uikit/uiapplication/1622952-canopenurl](https://developer.apple.com/documentation/uikit/uiapplication/1622952-canopenurl)